sympy/core/relational.py

```diff
@@ -1138,3 +1047,309 @@ def _eval_relation(cls, lhs, rhs):
+    using multiple dispatch
+
+    Examples
+    ========
+
+    >>> from sympy.core.relational import is_ge
+    >>> from sympy.core.sympify import _sympify
+    >>> is_ge(_sympify(2), _sympify(0))
+    True
+
+    >>> from sympy.core.relational import is_ge
+    >>> from sympy.core.sympify import _sympify
+    >>> is_ge(_sympify(0), _sympify(2))
+    False
+
+    >>> from sympy.abc import x
+    >>> from sympy.core.relational import is_ge
+    >>> from sympy.core.sympify import _sympify
+    >>> is_ge(_sympify(0), x)
+
+    >>> from sympy.core.relational import is_gt
+    >>> from sympy.core.sympify import _sympify
+    >>> is_gt(_sympify(2), _sympify(0))
+    True
+
+    >>> from sympy.core.relational import is_gt
+    >>> from sympy.core.sympify import _sympify
+    >>> is_gt(_sympify(0), _sympify(2))
+    False
+
+    >>> from sympy.core.relational import is_lt
+    >>> from sympy.core.sympify import _sympify
+    >>> is_lt(_sympify(0), _sympify(2))
+    True
+
+    >>> from sympy.core.relational import is_lt
+    >>> from sympy.core.sympify import _sympify
+    >>> is_lt(_sympify(2), _sympify(0))
+    False
+
+   """
+    if not (isinstance(lhs, Expr) and isinstance(rhs, Expr)):
+        raise TypeError("Can only compare inequalities with Expr")
+
+    retval = _eval_is_ge(lhs, rhs)
+
+    if retval is not None:
+        return retval
+    else:
+        n2 = _n2(lhs, rhs)
+        if n2 is not None:
+            # use float comparison for infinity.
+            # otherwise get stuck in infinite recursion
+            if n2 in (S.Infinity, S.NegativeInfinity):
+                n2 = float(n2)
+            return _sympify(n2 >= 0)
+        if lhs.is_extended_real and rhs.is_extended_real:
+            if (lhs.is_infinite and lhs.is_extended_positive) \
+                or (rhs.is_infinite and rhs.is_extended_negative):
+                return True
+            diff = lhs - rhs
+            if diff is not S.NaN:
+                rv = diff.is_extended_nonnegative
+                if rv is not None:
+                    return rv
+
+
+def is_neq(lhs, rhs):
+    """Fuzzy bool for lhs does not equal rhs.
+
+    See the docstring for is_eq for more
+    """
+    return fuzzy_not(is_eq(lhs, rhs))
+
+
+def is_eq(lhs, rhs):
+    """
+    Fuzzy boolean used to check if lhs is equal, mathematically
+    to rhs.
+    Returns True or False if the two are comparable, or None if the two are incomparable
+    or if the comparison is indeterminate.
+
+     To override the default equality logic, use _eval_is_eq with multiple dispatch.
+     is_ne, fuzzy nots is_eq so, to override the logic of is_ne, you will need to override _eval_is_eq
+
+    Examples
+    ========
+
+    >>> from sympy.core.relational import is_eq
+    >>> from sympy.core.sympify import _sympify
+    >>> is_eq(_sympify(0), _sympify(0))
+    True
+
+    >>> from sympy.core.relational import is_neq
+    >>> from sympy.core.sympify import _sympify
```

Comment:
It isn't necessary to reimport a function that has already been imported by a doctest in the same docstring. It's better just to put all imports at the top and then show the examples with some explanation. It's also worth showing how `a == b` and `Eq(a, b)` would behave with similar arguments.

For user documentation we wouldn't normally use `_sympify` rather than `sympify`. In fact `S` is fine e.g. `S(0)`.